testsrc/main/mondrian/xmla/XmlaBasicTest.java

```diff
@@ -744,12 +744,12 @@ public void testExecuteAliasWithSharedDimension()
             + "<Schema name=\"foodmart-xmla-alias-bug\">\n"
             + "  <Dimension name=\"Customers\">\n"
             + "    <Hierarchy hasAll=\"true\" allMemberName=\"All Customers\" primaryKey=\"customer_id\">\n"
-            + "      <Table schema=\"foodmart\" name=\"customer\"/>\n"
+            + "      <Table name=\"customer\"/>\n"
```

Comment:
@mchen-len-son Why does removing the schema attribute fix the tests?  Can you provide a little background on what was failing and why?